config/formats.ts

```diff
@@ -2504,7 +2507,7 @@ export const Formats: import('../sim/dex-formats').FormatList = [
 		name: "[Gen 8] National Dex Monotype",
 		mod: 'gen8',
 		searchShow: false,
-		ruleset: ['Standard NatDex', 'Same Type Clause', 'Species Clause', 'OHKO Clause', 'Evasion Moves Clause', 'Evasion Items Clause', 'Dynamax Clause', 'Sleep Clause Mod'],
+		ruleset: ['Standard NatDex', 'Same Type Clause', '!Evasion Clause', 'Evasion Moves Clause', 'Evasion Items Clause', 'Dynamax Clause'],
```

Comment:
```suggestion
		ruleset: ['Standard NatDex', 'Same Type Clause', '!Evasion Abilities Clause', 'Dynamax Clause'],
```

Reply:
Evasion Clause also contains the message:

> Evasion Clause: Evasion abilities, items, and moves are banned

Which would still appear if you did it this way.